docs/Contributing.asciidoc

```diff
@@ -294,6 +294,10 @@ assets can need a big amount of disk space.
 WARNING: Be sure to *clear* that variable when running unit tests locally.
 
 === Customize configuration directory
+When running openQA from a Git checkout it will find configuration files from
```

Comment:
Please add documentation for admins, not just developers. Maybe a simple sentence is enough but in the according section

Reply:
So far nothing has changed for admins. The config files are still under `/etc`. I will extend the config for admins as soon as we I make changes to install/package config files under `/usr/etc`.